### PR TITLE
[Storybookv8]Add files for Send Box

### PR DESCRIPTION
**Pull request by AmyL219**
# What
Screenshot 2024-07-03 at 9 47 33 AM
Screenshot 2024-07-03 at 9 47 26 AM
Screenshot 2024-07-03 at 9 47 20 AM

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/storybook8/stories/Components/SendBox/SendBox.mdx [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import { SendBox } from '@azure/communication-react';
```
Comment (JoshuaLai) [inline packages/storybook8/stories/Components/SendBox/SendBox.mdx:1]: As discussed update this name to match the other as `Doc.mdx`

File: packages/storybook8/stories/Components/SendBox/SendBox.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,101 @@
+// Copyright (c) Microsoft Corporation.
```
Comment (JoshuaLai) [inline packages/storybook8/stories/Components/SendBox/SendBox.stories.tsx:1]: As discussed renaming this file to match the other as `index.stories.tsx`

File: packages/storybook8/stories/Components/SendBox/SendBox.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+          window.alert(`onCancelAttachmentUpload callback is called for attachment id: "${attachmentId}"`);
+        }}
+      />
+    </div>
+  );
+};
+
+Default.storyName = 'SendBox';
+
+export default {
```
Comment (JoshuaLai) [inline packages/storybook8/stories/Components/SendBox/SendBox.stories.tsx:56]:
```suggestion
const meta: Meta<typeof SendBoxComponents>
```

File: packages/storybook8/stories/Components/SendBox/SendBox.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+  title: 'Components/SendBox/Send Box',
```
Comment (JoshuaLai) [inline packages/storybook8/stories/Components/SendBox/SendBox.stories.tsx:57]: I think in this case we don't need to put Send Box under a folder. 
image

As per the new storybook v8 I think it just needs to be `'Components/Send Box'`
And RichTextSendBox will also sit under `'Components/Rich Text Send Box'`

Comment (AmyL219) [reply packages/storybook8/stories/Components/SendBox/SendBox.stories.tsx]: But this is what we are doing right now in our current Storybook. Keep `SendBox` and `RichTextSendBox` under same folder. They re related to each other and should be grouped together.

File: packages/storybook8/stories/Components/SendBox/Docs.mdx [hunk 5]
```diff
@@ -0,0 +1,72 @@
+import { SendBox } from '@azure/communication-react';
+import { Title, Description, Props, Heading, Source, Canvas, Meta, ArgsTable } from '@storybook/addon-docs';
+import { SingleLineBetaBanner } from '../../BetaBanners/SingleLineBetaBanner';
+import { DetailedBetaBanner } from '../../BetaBanners/DetailedBetaBanner';
+import { AttachmentUploadsExample } from './snippets/AttachmentUploads.snippet';
+import { CustomIconExample } from './snippets/CustomIcon.snippet';
```
Comment (prabhjot-msft) [inline packages/storybook8/stories/Components/SendBox/Docs.mdx:6]: These snippet imports can be removed from here, as they are only used in index.stories.ts

File: packages/storybook8/stories/Components/SendBox/index.stories.tsx [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+    hasAttachments: controlsToAdd.isSendBoxWithAttachments,
+    systemMessage: hiddenControl,
+    onSendMessage: hiddenControl,
+    onTyping: hiddenControl,
+    onRenderSystemMessage: hiddenControl,
+    supportNewline: hiddenControl,
+    onRenderIcon: hiddenControl,
+    styles: hiddenControl,
+    strings: hiddenControl
+  }
```
Comment (prabhjot-msft) [inline packages/storybook8/stories/Components/SendBox/index.stories.tsx:58]: Default values can be added here as `args:{}`

File: packages/storybook8/stories/controlsUtils.ts [hunk 7]
```diff
@@ -269,6 +269,7 @@ export const controlsToAdd = {
   isScreenSharing: { control: 'boolean', defaultValue: false, name: 'Is screen sharing' },
   isRaisedHand: { control: 'boolean', defaultValue: false, name: 'Is Raised Hand' },
   isSendBoxWithWarning: { control: 'boolean', defaultValue: false, name: 'Has warning/information message' },
+  isSendBoxWithAttachments: { control: 'boolean', defaultValue: false, name: 'Has attachments' },
```
Comment (prabhjot-msft) [inline packages/storybook8/stories/controlsUtils.ts:272]: Remove the default value from here

**Review by JoshuaLai — APPROVED** [review]

**Review by prabhjot-msft — APPROVED** [review]

**Pull request merged**